docs/SPEC.md

```diff
@@ -1903,6 +1903,11 @@ from(bucket:"telegraf/autogen")
                 r.service == "app-server")
 ```
 
+##### Helper Predicate Functions
+Two predicate functions that return boolean values and are useful in filters are: 
+* `contains(value:T0, set: [T0...])`: tests whether the `set` contains the given value.  The type of value and set can vary, but must match.
+* `exists(r:R0, key: "key")`: tests whether the record `r` has a field `key`.  
+
```

Comment:
Contains was already documented in the SPEC. Can you move its doc to this section of the SPEC? And give `exists` its own heading as well? This should make them easier to find.

Reply:
It looks like we are inconsistent in referring to object keys are either keys or properties. The grammar referres to them as a `property` and the text of the SPEC occasionally uses `key`. Lets standardize on one, I vote for `property` as that is what the grammar uses.

Reply:
I've always thought of it like "a property is composed of a key/value pair."

Reply:
yes, I must have  missed the other reference to contains.  will fix.

Reply:
That makes sense, perhaps we should write that phrase at least once in the SPEC to make the association?